t/integration/test_canvas.py

```diff
@@ -3132,12 +3132,12 @@ def task_received_handler(request=None, **kwargs):
                     [
                         stamped_header in link.options
                         for stamped_header in link.options["stamped_headers"]
-                        if link  # the link itself doensn't have a link
+                        if link  # the link itself doesn't have a link
                     ],
                     [
                         stamped_header in link_error.options
                         for stamped_header in link_error.options["stamped_headers"]
-                        if link_error  # the link_error itself doensn't have a link
+                        if link_error  # the link_error itself doesn't have a link
```

Comment:
```suggestion
                        if link_error  # the link_error itself doesn't have a link_error
```
This might also be a copy past typo, but since I'm still trying to understand the code I didn't change it.